paddle/operators/math/CMakeLists.txt

```diff
@@ -3,11 +3,15 @@ if(WITH_GPU)
     nv_test(math_function_test SRCS math_function_test.cc DEPS math_function tensor)
     nv_library(softmax SRCS softmax.cc softmax.cu DEPS operator)
     nv_library(cross_entropy SRCS cross_entropy.cc cross_entropy.cu DEPS operator)
+    nv_library(vol2col SRCS vol2col.cc vol2col.cu DEPS device_context operator)
```

Comment:
Why depend on `operator` when compiling in this function?

Reply:
Done